tools/config-generator/testgrid_config.go

```diff
@@ -168,37 +170,75 @@ func getGcsLogDir(testGroupName string) string {
 	return fmt.Sprintf("%s/%s/%s", GCSBucket, LogsDir, testGroupName)
 }
 
+func getTestgroupExtras(projName, jobName string) map[string]string {
+	extras := make(map[string]string)
+	switch jobName {
+	case "continuous":
+		// projName has the release encoded into it, so the main page at http://testgrid.knative.dev
+		// does not mix releases with the master branch
+		if releaseRegex.FindString(projName) != "" {
+			extras["num_failures_to_alert"] = "3"
```

Comment:
I think you need to quote these numbers like `"\"3\""`, this seems to be the reason that the Prow job config check job is failing.

Reply:
Unquoted is correct within this method since it writes the Testgrid YAML ([example](https://github.com/knative/test-infra/blob/8f5dc4814ac37f9d504bd6872673a7d1bdbb0f5d/config/prod/prow/testgrid/testgrid.yaml#L55), but quoted numbers are required below in the generateProwJobAnnotations method and have been updated.